### ISSUE TITLE
Use git tags that sort numerically to the latest version (e.g. YYMMDD).

### ISSUE THREAD

**Issue by kanavin** [open-acpica/acpica#886]
Currently the tags are in MMDDYY format, and so it's impossible to determine what the latest version is (or what is the general order of releases), unless one wants to write a custom script to parse the tags. 

https://github.com/acpica/acpica/tags

Typically versions can be numerically compared and are thus sortable.

**Comment by dvzrv** [comment]
Hi! I'm currently looking into updating the package for this project on Arch Linux.

@acpibob, please switch to e.g. a [calver](https://calver.org/) tagging scheme (as date based versioning is seemingly what you are doing here anyways) going forward (e.g. today == `20230829`, which would still allow you to release multiple times per day by appending a minor version, e.g. `20230829.1`) going forward (you can also drop the leading `R`, as it serves no purpose).

It would also be ideal if you could create a release following this new tagging scheme right away, since downstreams are by now barred from downloading the source code from the intel website at https://www.intel.com/content/www/us/en/download/776307/acpi-component-architecture-downloads-unix-format-source-code-and-build-environment-with-a-dual-license.html in an automatable fashion (which is required for packaging and reproducible builds purposes), since the downloads moved there.

On a somewhat related note: It would be great if github releases could be facilitated to offer the source tarballs, since github's auto-generated tarballs will become unstable (the compression algorithm will change in 2024, breaking their currently stable checksums). This is required for downstreams to be able to use this project to build packages from the sources (locking the checksums of the upstream tarballs).

**Comment by andy-shev** [comment]
You can add as many tags per the same commit as you want. So, I propose to keep an old schema and add a new one. It's quite easy to go through all tags and apply a alternative format **for all of them**.

**Comment by SaketADumbre** [comment]
I believe this issue is already serviced, not to mention that we have been doing GitHub releases for quite a while now and also following an updated YYYYMMDD CalVer formatting for release tags! As for the Intel website issue and also concerning people's broken scripts, I am working to address that now and will issue a statement on the rationale behind the website move, plans moving forward and closing out all the duplicates here in the Issues section of ACPICA! Thanks a lot once again for your patience and we appreciate any additional feedback you may have regarding virtually any topic related to ACPICA!

**Comment by kanavin** [comment]
> I believe this issue is already serviced, not to mention that we have been doing GitHub releases for quite a while now and also following an updated YYYYMMDD CalVer formatting for release tags!

Not quite. The two latest tags are R2025_04_04 and R2024_12_12. That at least puts the year in front, but it is not YYYMMDD and doesn't match the tarballs (which do use YYYYMMDD). If you have to keep the RYYYY_MM_DD scheme, I'd suggest using additional tags which can be matched to tarball versioning more easily.

**Comment by SaketADumbre** [comment]
OK I hear your thoughts @kanavin and I can ask the community with a poll on how to name our release tags, but I am guessing that the most simple CalVer format YYYYMMDD would win and I would support matching the tarballs/zipfiles in the release.

**Comment by maciejwieczorretman** [comment]
I see that since august 2025 the tags are in numerically sortable format. Closing this issue
